metrics/prometheus/prometheus.go

```diff
@@ -58,25 +58,35 @@ type Metrics struct {
 	adsCertSignTimer             prometheus.Histogram
 
 	// Adapter Metrics
-	adapterBids                *prometheus.CounterVec
-	adapterErrors              *prometheus.CounterVec
-	adapterPanics              *prometheus.CounterVec
-	adapterPrices              *prometheus.HistogramVec
-	adapterRequests            *prometheus.CounterVec
-	adapterRequestsTimer       *prometheus.HistogramVec
-	adapterReusedConnections   *prometheus.CounterVec
-	adapterCreatedConnections  *prometheus.CounterVec
-	adapterConnectionWaitTime  *prometheus.HistogramVec
-	adapterGDPRBlockedRequests *prometheus.CounterVec
+	adapterBids                           *prometheus.CounterVec
+	adapterErrors                         *prometheus.CounterVec
+	adapterPanics                         *prometheus.CounterVec
+	adapterPrices                         *prometheus.HistogramVec
+	adapterRequests                       *prometheus.CounterVec
+	adapterRequestsTimer                  *prometheus.HistogramVec
+	adapterReusedConnections              *prometheus.CounterVec
+	adapterCreatedConnections             *prometheus.CounterVec
+	adapterConnectionWaitTime             *prometheus.HistogramVec
+	adapterGDPRBlockedRequests            *prometheus.CounterVec
+	adapterBidResponseValidationSizeError *prometheus.CounterVec
+	adapterBidResponseValidationSizeWarn  *prometheus.CounterVec
+
+	adapterBidResponseSecureMarkupError *prometheus.CounterVec
+	adapterBidResponseSecureMarkupWarn  *prometheus.CounterVec
```

Comment:
Newly created `CounterVec`s need to be preloaded, inside `metrics/prometheus/preload.go`. For instance `storedAccountErrors`:

```
38       storedAccountErrors          *prometheus.CounterVec
metrics/prometheus/prometheus.go
```

Gets preloaded like:

```
84     preloadLabelValuesForCounter(m.storedAccountErrors, map[string][]string{
85         storedDataErrorLabel: storedDataErrorValues,
86     })
metrics/prometheus/preload.go
```

We need to do the same for `adapterBidResponseValidationSizeError`, `adapterBidResponseValidationSizeWarn`, `adapterBidResponseSecureMarkupError`, and `adapterBidResponseSecureMarkupWarn`